luadox/render/html.py

```diff
@@ -283,10 +288,18 @@ def _render_html(self, topref: TopRef, lines: List[str]) -> Generator[
         root = self._get_root_path()
         head: list[str] = []
 
-        css = self.config.get('project', 'css', fallback=None)
-        if css:
+        css_files = self.config.get('project', 'css', fallback='').split()
+        for i, css in enumerate(css_files):
```

Comment:
We aren't using the index `i` where so enumerate() isn't necessary.  So just the simpler `for css in css_files` form.

Or, combined with the utility function mentioned above:

```python
for css in files_str_to_list(self.config.get('project', 'css', fallback='')):
    # ...
```